packages/calcite-components/src/components/tile/tile.scss

```diff
@@ -202,21 +233,28 @@
 
 :host([selection-appearance="border"][layout="horizontal"]) {
   .container.selected {
-    box-shadow: inset 0 -4px 0 0 var(--calcite-color-brand);
+    --calcite-internal-tile-shadow: inset 0 -4px 0 0 var(--calcite-color-brand);
   }
 }
 
 :host([selection-appearance="border"][layout="vertical"]) {
   .container.selected {
-    box-shadow: inset 4px 0 0 0 var(--calcite-color-brand);
+    --calcite-internal-tile-shadow: inset 4px 0 0 0 var(--calcite-color-brand);
+  }
+}
+
+:host([href]:active:not([disabled])) {
+  .container {
+    box-shadow: 0 0 0 3px var(--calcite-tile-border-color);
   }
 }
 
 :host([href]:focus:not([disabled])),
 :host([href]:hover:not([disabled])) {
-  --calcite-tile-border-color: var(--calcite-color-text-link);
-  --calcite-tile-heading-text-color: var(--calcite-color-text-link);
-  --calcite-icon-color: var(--calcite-color-text-link);
+  --calcite-internal-tile-border-color: var(--calcite-color-text-link);
+  --calcite-internal-tile-heading-text-color: var(--calcite-color-text-link);
+  --calcite-internal-tile-icon-color: var(--calcite-color-text-link);
```

Comment:
I think we need to add an extra token for this case `--calcite-title-href-text-color`
```css
:host([href]:focus:not([disabled])),
:host([href]:hover:not([disabled])) {
  .container,
  .heading {
    color: var(--calcite-title-href-text-color, var(--calcite-color-text-link));
  }
  .container {
    border-color: var(--calcite-title-href-text-color, var(--calcite-color-text-link));
  }
}
```